tests/python/frontend/pytorch/test_forward.py

```diff
@@ -23,6 +23,7 @@
 from scipy.stats import t as tdistr
 import numpy as np
 import torch
+from torch import nn
```

Comment:
Nitpick: There's places where we use torch.nn and others where we use nn, can we change the former to the latter and remove the below import as well.

Reply:
Removed `from torch import nn` and updated to use `torch.nn`